chart/kubeapps/templates/dashboard-config.yaml

```diff
@@ -16,9 +16,71 @@ data:
       gzip on;
       gzip_static  on;
 
-      location / {
+      # /apps/ns/:namespace
+      location ~ "/apps/ns/[^/]*/?$" {
```

Comment:
Ok so this is so the server returns a proper 404 status code and not a 200 no?

Why is that that important? If this was an API I'd think differently, but for a SPA, the fact that it returns 200 does not seem a big deal as long as the user is presented with a not found page.

I think that the tradeoff of maintaining these routes in both places is too much to ask for that 404 improvement.

Reply:
okay, I agree